docs/sources/tempo/api_docs/_index.md

```diff
@@ -340,7 +360,8 @@ $ curl -G -s http://localhost:3200/api/v2/search/tags  | jq
 
 ### Search tag values
 
-Ingester configuration `complete_block_timeout` affects how long tags are available for search.
+Ingester configuration `complete_block_timeout` affects how long tags are available for search, if start or end are not specified it will only
+fetch blocks that wasn't flushed to backend.
```

Comment:
```suggestion
Ingester configuration `complete_block_timeout` affects how long tags are available for search. if start or end are not specified, it will only
fetch blocks that wasn't flushed to backend.
```